.github/workflows/dependencies.yml

```diff
@@ -9,8 +9,11 @@ on:
 
 jobs:
   community_check:
+    if: github.event_name == 'pull_request_target'
```

Comment:
See comment on `changelog.yml`.

Reply:
In looking at this further, the `comment` action uses `github.event.pull_request.number`, which is not included in `push` events. I'm hesitant to remove the `on.push` triggers from this one, since the `go_mod` job would still run for those events, though I'm unsure of the utility of that job at the time of writing.